baseband/gsb/payload.py

```diff
@@ -94,20 +98,22 @@ def fromfile(cls, fh, payloadsize=None, bps=4, nchan=1,
             Number of bits per sample part (i.e., per channel and per real or
             imaginary component).  Default: 4.
         nchan : int
-            Number of fourier channels.  Default: 1.
+            Number of Fourier channels.  Default: 1.
         complex_data : bool
             Whether data is complex or float.  Default: False.
         """
         if hasattr(fh, 'read'):
+            sample_shape = cls._sample_shape_cls_1thread(nchan)
```

Comment:
If we go with the above, you'd do `sample_shape=(nchan,)` here.

Reply:
Changed.